examples/tutorial/tutorial_notebooks/cluster/exercise_5.ipynb

```diff
@@ -130,40 +102,29 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "ray.shutdown()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Shutting down the cluster\n",
+    "# DO NOT CHANGE THIS CODE!\n",
+    "# Changing this code risks breaking further exercises\n",
     "\n",
-    "**You may have to change the path below**. If this does not work, log in to your \n",
-    "\n",
-    "Now that we have finished computation, we can shut down the cluster with `ray down`."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "!ray down modin-cluster.yaml"
+    "import time\n",
+    "time.sleep(600) # We need to give ray enough time to start up all the workers\n",
```

Comment:
Is this correct? This is a very long wait time to get the cluster to start running.

Reply:
It may be less now, but in the past this was necessary. Most of the wait time is aws actually provisioning the resources.